tgui-next/packages/tgui/interfaces/MedicalKiosk.js

```diff
@@ -0,0 +1,151 @@
+import { Fragment } from 'inferno';
+import { act } from '../byond';
+import { AnimatedNumber, Box, Button, LabeledList, ProgressBar, Section} from '../components';
+
+export const MedicalKiosk = props => {
+  const { state } = props;
+  const { config, data } = state;
+  const { ref } = config;
+ 
+  return (
+    <Fragment>
+      <Section title="Health Kiosk"
+        textAlign="center">
+        <Box as="span" m={1} textAlign="center">
+          Greetings Valued Employee. Please select your desired treatment type from the options below.
+          <Button
+            icon="procedures"
+            disabled={data.active_status}
+            onClick={() => act(ref, 'beginScan')}
+            content="General Health Scan" />
+          <Button
+            icon="heartbeat"
+            disabled={data.active_status}
+            onClick={() => act(ref, 'beginScan')}
+            content="Symptom Based Checkup" />
+          <Button
+            icon="radiation-alt"
+            disabled={data.active_status}
+            onClick={() => act(ref, 'beginScan')}
+            content="Neurological/Radiological Scan" />
+        </Box>
+      </Section>
+      <Section title="Patient Health"
+        textAlign="center">
+        <LabeledList>
+          <LabeledList.Item
+            label="Total Health">
+            <ProgressBar
+              value={data.patient_health/100}
+              color="grey">
+              <AnimatedNumber value={data.patient_health} />%
+            </ProgressBar>
+            <LabeledList.Divider size={2} />
```

Comment:
Divider should be placed at the same level as Item.

I'm surprised this even works, because `<LabeledList.Item>` is a `<tr><td>` internally, and `<LabeledList.Divider>` is a `<tr><td>` internally, so you get a final markup as `<tr><td><tr><td></td></tr></td></tr>`.